chart/flux/values.yaml

```diff
@@ -312,3 +312,7 @@ hostAliases: {}
 #    hostnames:
 #    - "foo.remote"
 #    - "bar.remote"
+
+dashboards:
+  # If enabled, flux will create a configmap with a dashboard in json that's going to be picked up by grafana
+  enabled: true
```

Comment:
Add a new line here. Also please set it to false as this depends on prom annotations that are disabled by default.